src/events.c

```diff
@@ -295,7 +298,10 @@ static void feh_event_handle_ButtonPress(XEvent * ev)
 				- (winwid->im_click_offset_y * winwid->zoom);
 
 		winwidget_sanitise_offsets(winwid);
-		winwidget_render_image(winwid, 0, 0);
+		if(opt.antialiasing_zoom == 0)
```

Comment:
This block is copied too many times, make it a function maybe?

Reply:
```suggestion
		if (opt.antialiasing_zoom == 0)
```
And so on for all the rest to match the existing style.